internal/k8s/portforward.go

```diff
@@ -124,26 +124,20 @@ func (c portForwardClient) CreatePortForwarder(ctx context.Context, namespace Na
 
 	ports := []string{fmt.Sprintf("%d:%d", localPort, remotePort)}
 
-	var pf *portforward.PortForwarder
 	if host == "" {
-		pf, err = portforward.New(
```

Comment:
I was not -- tried disabling ipv6 on the loopback on macos and the described behavior doesn't happen. It looks like an issue specific to ipv6 disabled on Windows.

Reply:
Yeah, had the same thought about `kubectl port-forward`, waiting for feedback on it.

Reply:
ok, i think i understand the problem, i think you've got the fixed reversed.

portforward.go special-cases `localhost` - if you pass it `localhost`, it starts the port-forwarders in a special failureMode. As long as one of the binds succeeds, the port-forward succeeds.

my theory what's happening is that on this user's machine, `localhost` resolves to ipv4 and ipv6 even though they have ipv6 turned off. `getListenableAddresses` resolves, circumventing the special-case logic in `kubectl` / `client-go`

possible fixes:
- add additional special-cases in `client-go` to set failureMode: 'all' for loopback ips
- special-case getListenableAddresses so that `localhost` isn't resolved
wdyt?

still want to verify manually on a windows box